src/pages/attribute-dictionary.js

```diff
@@ -34,7 +35,7 @@ const AttributeDictionary = ({ data, pageContext }) => {
     [allDataDictionaryEvent]
   );
 
-  const filterEvents = () => {
+  useEffect(() => {
```

Comment:
I assume this is a change that doesn't have to do with the localized links?

Reply:
eslint was yelling at me telling me that `filterEvents` was missing as a dependency. Because of how hooks work with regards to closures, I merge the `useEffect` and made its function the `filterEvents` function.

You're right though, doesn't have to do with the localized links.

Reply:
Ah okay! Just checking, wanted to make sure I wasn't missing some special property of the localized links :)